src/libs/SidebarUtils.ts

```diff
@@ -104,6 +105,12 @@ function getOrderedReportIDs(
             return false;
         }
 
+        const participantAccountIDs = Object.keys(report?.participants ?? {}).map(Number);
+
+        if (currentUserAccountID && AccountUtils.isAccountIDOddNumber(currentUserAccountID) && participantAccountIDs.includes(CONST.ACCOUNT_ID.NOTIFICATIONS)) {
```

Comment:
Hey team, coming from https://github.com/Expensify/App/issues/43782
This early return returns `true` (displays in LHN) for **all** chats that have `CONST.ACCOUNT_ID.NOTIFICATIONS` as a participant. 
Domain chat room does have `CONST.ACCOUNT_ID.NOTIFICATIONS` as a participant, but doesn't need to be displayed in focus mode, since there are no unread messages

Reply:
@eVoloshchak Should I fix it or you're going with someone's proposal?

Reply:
@koko57, I'm not entirely sure
On the one hand, this PR was deployed to prod 5 days ago, it's still in 7-day regression period, so it makes sense  to fix it here
On the other hand, we've opened up the issue to contributors and have already received valid proposals

@twisterdotcom, could you weigh in on this please?

Reply:
@koko57 yes please, lets fix it, its still in the regression period

Reply:
Similar issue was detected where expensify chat showed up in focus mode even though there were no unread messages https://github.com/Expensify/App/issues/43599. This was fixed in https://github.com/Expensify/App/pull/44061